bindings/python/dlite-collection-python.i

```diff
@@ -56,19 +56,24 @@ class Collection(Instance):
     to it.  Likewise, the len() of a collection will return the number
     of instances.
 
-    Use the get_relations() method (or the convenience methods
-    get_subjects(), get_predicates() and get_objects()) to iterate
+    Use the `get_relations()` method (or the convenience methods
+    `get_subjects()`, `get_predicates()` and `get_objects()`) to iterate
     over relations.  The number of relations is available via the
     `nrelations` property.
 
-    Relations are (s, p, o, d=None)-triples with an optional fourth field
+    Relations are `(s, p, o, d=None)`-triples with an optional fourth field
     `d`, specifying the datatype of the object.  The datatype may have the
     following values:
 
-      - None: object is an IRI.
-      - Starts with '@': object is a language-tagged plain literal.
-        The language identifier follows the '@'-sign.
-      - Otherwise: object is a literal with datatype `d`.
+    - None: object is an IRI.
+    - Starts with '@': object is a language-tagged plain literal.
+      The language identifier follows the '@'-sign. e.g. `text@en`
```

Comment:
This is not clear. 
Do we write ´o='text', d='@en'´ or ´d='test@en´? As I suggested in my previous comment I think my example is not completely correct.

Reply:
You are right. It should be ´o='text', d='@en'´. Corrected the example